src/lib.rs

```diff
@@ -28,6 +29,7 @@
 //! [intel mkl]: https://software.intel.com/en-us/mkl
 //! [netlib]: http://www.netlib.org/
 //! [openblas]: http://www.openblas.net/
+//! [R]: https://cran.r-project.org
```

Comment:
The same comments as in the other repository. This has to be used in the list above and in the readme.